tracer/src/Datadog.Trace/Vendors/StatsdClient/Transport/UnixDomainSocketTransport.cs

```diff
@@ -29,7 +29,8 @@ internal class UnixDomainSocketTransport : ITransport
 
             try
             {
-                _socket = new Socket(AddressFamily.Unix, SocketType.Dgram, ProtocolType.Unspecified);
```

Comment:
Just putting a reminder here that we can't merge this because it's a change inside vendored code

Reply:
Fixed in cbb76250b0413b828b3fadfa8c9b56355a4a8686